base/client.jl

```diff
@@ -71,6 +72,17 @@ function repl_cmd(cmd, out)
         ENV["OLDPWD"] = new_oldpwd
         println(out, pwd())
     else
+        @windows_only begin
+            cmdbuf = IOBuffer()
+            if lowercase(shell[1]) == "cmd"
+                print_cmdshell_escaped(cmdbuf, cmd.exec...)
+                cmd.exec = ["cmd"; "/s /c \"$(bytestring(cmdbuf.data))\""]
```

Comment:
i think you can call `bytestring` on `cmdbuf` directly (and get the same result)